packages/dropdowns/src/styled/items/StyledItemMeta.ts

```diff
@@ -22,8 +22,8 @@ export const StyledItemMeta = styled.span.attrs({
   'data-garden-id': COMPONENT_ID,
   'data-garden-version': PACKAGE_VERSION
 })<IStyledItemMetaProps>`
-  display: ${props => (props.isCompact ? 'none' : 'block')};
-  line-height: ${props => props.theme.space.base * 4}px;
+  display: block;
+  line-height: ${props => props.theme.space.base * (props.isCompact ? 3 : 4)}px;
```

Comment:
This is one of the edge cases where Garden uses pixels rather than unitless `line-height` in order to account for observed browser (notably, Chrome) anomalies in menu item heights.

Reply:
Looks great! Ship it!